web/src/app/admin/indexing/status/CCPairIndexingStatusTable.tsx

```diff
@@ -68,6 +68,28 @@ function CCPairIndexingStatusDisplay({
   );
 }
 
+function ClickableTableRow({
+  url,
+  children,
+  ...props
+}: {
+  url: string;
+  children: React.ReactNode;
+  [key: string]: any; // This allows for any additional props
+}) {
+  const router = useRouter();
+
+  const navigate = () => {
+    router.push(url);
```

Comment:
When using `Link` from `next/link`, NextJS prefetches the data necessary to render the linked to page, which makes the navigation to the new page super fast. I don't believe this approach will do that 😓 It's not too big of a deal, but if possible it would be great to replicate that. My hunch is that there's a way to tell NextJS to prefetch a given url, which we could put in the `useEffect` of this component

Reply:
Added router.prefetch inside useEffect.